app/scripts/controllers/person/tabSubjects.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+angular.module('admissionSystemApp')
+  .controller('tabSubjects', ['$scope', '$modal', '$rootScope', 'DictionariesSvc', function ($scope, $modal, $rootScope, DictionariesSvc) {
+
+
+    // temp data --------begin---------
+    $scope.entirePerson.enrolmentsubjects = [
```

Comment:
Yeap(
Just to show during demo how my tab will look like when our restangular will receive data.